### ISSUE TITLE
PHP Email not working

### ISSUE THREAD

**Issue by rajil** [fusionpbx/fusionpbx#1931]
I set up the gmail parameters in Default Settings/Email. 
smtp_auth:true
smtp_from:a.b.c@gmail.com
smtp_from_name: PBX server
smtp_host:smtp.gmail.com
smtp_password:my real password
smtp_port:587
smtp_secure:tls
smtp_username:a.b.c@gmail.com (yes it has two dots in username)

The email error was
```
## less /tmp/mailer-app.log 
X-FusionPBX-Domain-UUID: 843ce8f5-57da-428e-b465-022679e43a98
X-FusionPBX-Domain-Name: xx.dyndns.org
X-FusionPBX-Call-UUID: 239ad232-d47d-e611-ab2d-0cc47adeccc0
X-FusionPBX-Email-Type: voicemail
Subject: =?utf-8?B?Vm9pY2VtYWlsIGZyb20gMjAxIDwyMDE+IDAwOjAwOjA0?=
From: a.b@gmail.com
Reply-to: 
To: a.b@gmail.com
Date: 
Add Address: a.b@gmail.com
SMTP -> FROM SERVER:220 smtp.gmail.com ESMTP tq5sm25871660pac.42 - gsmtp

SMTP -> FROM SERVER: 250-smtp.gmail.com at your service, [xx.xx.xx.xx]
250-SIZE 35882577
250-8BITMIME
250-STARTTLS
250-ENHANCEDSTATUSCODES
250-PIPELINING
250-CHUNKING
250 SMTPUTF8

SMTP -> FROM SERVER:220 2.0.0 Ready to start TLS

SMTP -> FROM SERVER:^U^C^A^@^B^BF
SMTP -> ERROR: RSET failed: ^U^C^A^@^B^BF
SMTP -> NOTICE:
EOF caught while checking if connectedThe following From address failed: a.b.c@gmail.com
Mailer Error: The following From address failed: a.b.c@gmail.com

Retained in v_emails as email_uuid = c15a644c-351c-49de-8ef1-c163d228ffda
```

**Comment by markjcrane** [comment]
This would be a phpmailer bug please report the here.
https://github.com/PHPMailer/PHPMailer/issues

**Comment by rajil** [comment]
The upstream version is 5.2.16 with some TLS fixes in [5.2.15] (https://github.com/PHPMailer/PHPMailer/blob/master/changelog.md). Fusionpbx is using the old version [5.2.14] (https://github.com/fusionpbx/fusionpbx/blob/0e73d0d3dca4da7e317776ee06f6a6cf75df4d1d/resources/phpmailer/changelog.txt) which likely is causing the problem.

**Comment by MrCoolLV** [comment]
best way to update?
